R/utilities-ratio-comparison.R

```diff
@@ -224,7 +257,7 @@ isSamePopulationInSets <- function(structureSets) {
   isReference <- sapply(structureSets, function(set) {
     set$simulationSet$referencePopulation
   })
-  referenceSet <- structureSets[isReference]
+  referenceSet <- structureSets[isReference][[1]]
```

Comment:
This was where the issue actually was.
Since `structureSets[isReference]` is an array of lists, `referenceSet$simulationSet$populationFile` was `NULL`.
Thus logical `isSamePopulation` was always `FALSE`, which led to always perform the Monte Carlo simulation even when the population files were the same